### DISCUSSION TITLE
MOES human presence sensor - ZSS-QY-HP (TS-0601)

### DISCUSSION THREAD

**Discussion by flux6663** [Koenkk/zigbee2mqtt#23197] [Devices]
Hello,

Following a recent purchase for my home automation system, I wanted to test this Moes presence sensor. However, it did not appear to be compatible, even though I have found other people who have successfully retrieved the data.

![image](https://github.com/Koenkk/zigbee2mqtt/assets/64163708/1c246eea-7013-44d3-beb9-7a83a6c231d2)

Do you have any suggestions for solving this problem?

Mathis

**Comment by XFNeo** [comment]
I have the same device and z2m tells that it is not supported. Please assist,

**Comment by thecifu** [comment]
Yo estoy en la misma situación
